media/css/firefox/privacy/products.scss

```diff
@@ -0,0 +1,298 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/emphasis-box';
+@import '../../protocol/components/fxa-form';
+
+.t-medium .mzp-l-content {
+    max-width: $content-xl;
+}
+
+// Underline active link in sub navigation.
+.privacy-products .c-sub-navigation-item.privacy-products > a {
+    text-decoration: underline;
```

Comment:
I'm worried it will be confusing for users to have something that is not a link underlined. Do you think we could move back towards the original design direction for the subnav?

<img width="355" alt="Screen Shot 2019-10-03 at 14 57 15" src="https://user-images.githubusercontent.com/854701/66167154-38d57080-e5ee-11e9-9566-7fc1dcbe7d52.png">

Reply:
I'm worried that what's in the design is even more confusing, since the bold link is indistinguishable (in it's meaning) when comparing to the (already bold) links in the main navigation. Jeniffer said an underline here is OK, but let me circle back and see if there's anything better we can do compared to either solution.